src/main/java/cms/gov/madie/measure/resources/AdminController.java

```diff
@@ -484,7 +484,7 @@ private Callable<MeasureTestCaseValidationReport> buildCallableForMeasureId(
     return () -> testCaseService.updateTestCaseValidResourcesWithReport(measureId, accessToken);
   }
 
-  @PutMapping("/unlock")
+  @PutMapping("/measures/test-cases/unlockAll")
```

Comment:
Since we're updating the endpoints, let's work towards making them consistent across entities.
```suggestion
  @DeleteMapping("/measures/test-cases/lock")
```
It is good practice to use nouns instead of verbs in REST endpoints. it's a little confusing here since "lock" can be both a noun and verb. If we think of it as a noun, then we can perform CRUD operations using REST verbs against it.

`PUT /measures/test-cases/{id}/lock`: Obtain a lock.
`DELETE /measures/test-cases/{id}/lock`: Remove the lock.
`PUT /measures/test-cases/locks`: Add a lock to all test-cases. AKA Lock all test cases. I doubt we'd do this.
`DELETE /measures/test-cases/locks`: Remove all locks from all test cases.

Reply:
@jkotanchik-SB this is done.